src/components/App/App.js

```diff
@@ -0,0 +1,28 @@
+require('normalize.css/normalize.css')
+require('./App.css')
+
+import React from 'react'
+import Navbar from '../Navbar/Navbar'
+import { Switch, Route } from 'react-router-dom'
+import Shop from '../../screens/Shop/Shop'
```

Comment:
hmm i will read that later, just from quick look i think it's bad idea the have two folders for components, like `components` and `styledComponents` merge them together 
also `data, images` will move to `lib`, drop `config` it should be all provided as env from webpack at build time.

hmm i think containers i also good name here ( it contains all child components needed for page/screen to be created ) my comments was just about that `containers` was more used name and can be more known ( but really not important ) at this stage 

also next i would update to webpack2 and clean the config stuff there